docs/contributing/how-to-label-an-issue.md

```diff
@@ -1,97 +1,99 @@
 ---
-title: How to label an issue
----
 
-## What are issue labels?
+title: किसी इशू को लेबल कैसे करें
+---
 
-Issue labels are a tool in GitHub that are used to group issues into one or more categories.
+## लेबल इशू क्या हैं?
 
-[Check out Gatsby's labels (and their descriptions)](https://github.com/gatsbyjs/gatsby/issues/labels)
+इशू लेबल GitHub में एक उपकरण है जो एक या अधिक कैटगोरी में ग्रुप के इशू के लिए उपयोग किया जाता है।
```

Comment:
`इशू के लिए उपयोग किया जाता है` => `इश्यूज को एक साथ ग्रुप करने के लिए उपयोग किया जाता है`

Reply:
`लिए उपयोग किया जाता है` => `लिए उपयोग किया जाता है।`

you missed the purnviram

Reply:
this is still not fixed. check line number 7.

and please do not mark this as resolved. I will mark it as resolved after verifying your fix